src/domElement.js

```diff
@@ -0,0 +1,120 @@
+import {
+  listsContainer, lists, selectedListId, listDisplayContainer,
+  listTitleElement, tasksContainer, taskTemplate, newTaskInput, newTaskDate, newTaskDescription,
+  newTaskPriority, newTaskForm,
+} from './storeData';
+import colorTasks from './color';
+import editTaskForm from './editTasks';
+import removeTask from './removeTask';
+
+function clearElement(element) {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+}
+
+function renderLists() {
+  lists.forEach((list) => {
+    const listElement = document.createElement('li');
+    listElement.innerText = list.name;
+    listElement.dataset.listId = list.id;
+    if (list.id === selectedListId) {
+      listElement.classList.add('active-list');
+    }
+    listsContainer.appendChild(listElement);
+  });
+}
+
+function editTask(task, label) {
+  editTaskForm();
+  newTaskInput.value = task.name;
+  newTaskDate.value = task.date;
```

Comment:
- [ ] Kindly change these and **all** other regular functions to es6 arrow function syntax.